### DISCUSSION TITLE
Exporting Part Field - Image Paramater

### DISCUSSION THREAD

**Discussion by imager2019** [inventree/InvenTree#5929] [Q&A] [answered]
I have been investigating Exporting and Importing and have found a field in the Parts that does not appear to be able to export/import. The field is Image. I am able to assign an image to a part, but cannot find a corresponding option to export Part information that has the Image information.

![image-capture](https://github.com/inventree/InvenTree/assets/49574724/87611acd-733f-4a23-b220-80c971752d09)

We are trying to use the Admin option and using the PART subsection and using the Parts export - but does not seem to be included in the export.

Is this something I have missed somewhere on how to do this, or is there a list of fields which are not exportable?

John

**Comment by SchrodingersGat** [comment, marked as answer]
@imager2019 currently not possible - but see this new PR - https://github.com/inventree/InvenTree/pull/5971

Will be available in `0.13.0`
